core/domain/state_domain.py

```diff
@@ -1545,24 +1546,28 @@ def _validate_item_selec_input(self, strict: bool = False) -> None:
                 selections.
             ValidationError. Answer choices are empty or duplicate.
         """
-        # Here we use cast because we are narrowing down the type from
-        # various allowed cust. arg types to 'int', and here we are sure
-        # that the type is always going to be int because 'ItemInputSelection'
-        # customization args always contains 'minAllowableSelectionCount' key
-        # with int type of values.
-        min_value = cast(
-            int,
-            self.customization_args['minAllowableSelectionCount'].value
-        )
-        # Here we use cast because we are narrowing down the type from
-        # various allowed cust. arg types to 'int', and here we are sure
-        # that the type is always going to be int because 'ItemInputSelection'
-        # customization args always contains 'maxAllowableSelectionCount' key
-        # with int type of values.
-        max_value = cast(
-            int,
-            self.customization_args['maxAllowableSelectionCount'].value
-        )
+        min_value = self.customization_args['minAllowableSelectionCount'].value
+        max_value = self.customization_args['maxAllowableSelectionCount'].value
+
+        if min_value is not None and max_value is not None:
+            # Here we use cast because we are narrowing down the type from
```

Comment:
Do we need two 6-line comments here? They distract the reader. I suggest leaving one copy of a comment and shrinking it to 1-2 lines max.

Reply:
Done